src/content/1.7/themes/getting-started/asset-management/_index.md

```diff
@@ -215,6 +215,12 @@ assets:
         path: assets/js/external-lib.js
         priority: 30
         position: bottom
+    all:
+      - id: external-cdn
+        path: //cdn-url.com/external-lib.js
+        priority: 100
+        position: bottom
+        server: remote
```

Comment:
Rhe introduction to this block states:

> "For example, if you want to add an external library on each page and a custom library on the Product page:"

Adding this example here will be confusing. It would be best to explain it separately: "If you want to include a library hosted in a different server..."

Reply:
You're totally right.
I've added a new block with an example of css file too.
Let me know if it's clearer now.